### PR TITLE
Reduce the PPP size

### PR DESCRIPTION
**Pull request by siqiang-ng**
Tried converting to pdf and got 3 pages instead of 2 (hard limit).

Can be merged now.

### CODE REVIEW

File: docs/team/siqiang-ng.md [hunk 1]
```diff
@@ -3,79 +3,68 @@ layout: page
...
 
-## Project: Projact
-
-Projact is a **project management app that helps NUS computing students to organise their fellow computing students' contacts and their teams' meeting links and tasks**. It is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI).
+### Project: Projact 
+<hr>
+Projact is a contact and project management app that helps NUS computing students to
+ organise their contacts and their projects' meeting links and tasks. 
 
 Given below are my contributions to the project.
```
Comment (yeo-kexin) [inline docs/team/siqiang-ng.md]: i think it's nicer if there's a space between Projact's description and given below...? but if there's really no space then nvm haha

**Review by yeo-kexin — COMMENTED** [review]
lgtm apart from comment

File: docs/team/siqiang-ng.md (hunk 1 above)
Comment (siqiang-ng) [reply docs/team/siqiang-ng.md]: Yep there is a space i thought? HAHA

Comment (siqiang-ng) [reply docs/team/siqiang-ng.md]: ok i realise what u mean, made the changes u can see

**Review by yeo-kexin — APPROVED** [review]
lgtm

File: docs/team/siqiang-ng.md [hunk 2]
```diff
@@ -3,79 +3,67 @@ layout: page
 title: Ng Si Qiang's Project Portfolio Page
 ---
 
-## Project: Projact
-
-Projact is a **project management app that helps NUS computing students to organise their fellow computing students' contacts and their teams' meeting links and tasks**. It is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI).
+## Project: Projact
```
Comment (yeo-kexin) [inline docs/team/siqiang-ng.md:6]: do you want to make it ### to save more space? but if it already fits then should be fine!

Comment (siqiang-ng) [reply docs/team/siqiang-ng.md:6]: It already fits haha. Previously I put ### but the horizontal line is gone and the space is gone between the first two para

**Pull request merged**